search_objects/src/search_objects.py

```diff
@@ -0,0 +1,25 @@
+import json
+import boto3
+from boto3.dynamodb.conditions import Attr, Or
+
+dynamodb = boto3.resource('dynamodb')
+table = dynamodb.Table('kutter-table')
+
+def search_objects_data(search, limit):
+    filter_expression = Or( Attr('creationDate').contains(search), Attr('originalFileName').contains(search))
+    response = table.scan(Limit = limit,FilterExpression = filter_expression) 
```

Comment:
when you filter for creation date i would expect that you can define a range

Reply:
will be tracked in issue https://github.com/Chufle/kutter/issues/69